managed-databases/postgresql-and-mysql/how-to/apply-maintenance.mdx

```diff
@@ -0,0 +1,50 @@
+---
+meta:
+  title: How to apply scheduled maintenance to a Database Instance
+  description: This page explains how to apply maintenance to a Database Instance
+content:
+  h1: How to apply scheduled maintenance to a Database Instance
+  paragraph: This page explains how to apply maintenance to a Database Instance
+tags: managed-database postgresql mysql database-instance maintenance
+dates:
+  validation: 2024-04-19
+  posted: 2024-04-19
+categories:
+  - managed-databases
+  - postgresql-and-mysql
+---
+
+From time to time your Scaleway Managed Databases have to undergo maintenance to make sure that your nodes are up-to-date and have all the tools necessary to maintain a healthy lifecycle. For example, your engine version might need to be upgraded to the latest available minor version, or certain patches might need to be implemented.
+
+These maintenance are set up, run, and scheduled by Scaleway, but you can select when to apply them to avoid interruptions during peak times.
```

Comment:
```suggestion
This maintenance is set up, run, and scheduled by Scaleway, but you can select when to apply it to avoid interruptions during peak times.
```

Reply:
```suggestion
These maintenance operations are set up, run, and scheduled by Scaleway, but you can select when to apply them to avoid interruptions during peak times.
```